desktop/cmp/filter/FilterChooser.ts

```diff
@@ -16,21 +16,39 @@ import {withDefault} from '@xh/hoist/utils/js';
 import {splitLayoutProps} from '@xh/hoist/utils/react';
 import classNames from 'classnames';
 import {isEmpty, sortBy} from 'lodash';
-import PT from 'prop-types';
-
+import {ReactElement} from 'react';
 import './FilterChooser.scss';
 
+export interface FilterChooserProps extends BoxProps<FilterChooserModel> {
+    /** True to focus the control on render. */
+    autoFocus?: boolean,
+    /** True to disable user interaction. */
+    disabled?: boolean,
+    /** True to show a "clear" button at the right of the control.  Defaults to true. */
+    enableClear?: boolean,
+    /** Icon to display inline on the left side of the input. */
+    leftIcon?: ReactElement,
+    /** Max-height of dropdown. Either a number in pixels or a valid CSS string, such as '80vh'. */
+    maxMenuHeight?: number|string,
+    /** Placement of the dropdown menu relative to the input control. */
+    menuPlacement?: 'auto'|'top'|'bottom',
+    /** Width in pixels for the dropdown menu - if unspecified, defaults to control width. */
+    menuWidth?: number,
+    /** Text to display when control is empty. */
+    placeholder?: string
+}
+
 /**
  * A Select based control for searching and choosing filters.
  * @see FilterChooserModel
  */
-export const [FilterChooser, filterChooser] = hoistCmp.withFactory({
+export const [FilterChooser, filterChooser] = hoistCmp.withFactory<FilterChooserProps>({
     model: uses(FilterChooserModel),
     className: 'xh-filter-chooser',
     render({model, className, ...props}, ref) {
         const [layoutProps, chooserProps] = splitLayoutProps(props),
             {inputRef, suggestFieldsWhenEmpty, selectOptions, unsupportedFilter, favoritesIsOpen} = model,
-            {autoFocus, enableClear, leftIcon, maxMenuHeight, menuPlacement, menuWidth} = chooserProps,
+            {autoFocus, enableClear, leftIcon, maxMenuHeight, menuPlacement, menuWidth} = chooserProps as FilterChooserProps,
```

Comment:
We can parameterize splitLayoutProps() so you don't need to do this -- I will take a look and circle back